plugins/outputs/warp10/warp10.go

```diff
@@ -0,0 +1,198 @@
+package warp10
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/influxdb/client/v2"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+type Warp10 struct {
+	Prefix string
+
+	WarpUrl string
+
+	Token string
+
+	Debug bool
+}
+
+var sampleConfig = `
+  # prefix for metrics class Name
+  prefix = "telegraf."
+  ## Telnet Mode ##
+  # Url name of the Warp 10 server
+  warpUrl = "localhost:4242/"
+  # Token to access your app on warp 10
+  token = "Token"
+  # Debug true - Prints Warp communication
+  debug = false
+`
+
+type MetricLine struct {
+	Metric    string
+	Timestamp int64
+	Value     string
+	Tags      string
+}
+
+func (o *Warp10) Connect() error {
+	// TODO Test Connection to Warp Server
```

Comment:
As it's an HTTP POST, I deleted this TODO for now. In fact, an error is raised when the push fails. I would like to know if it is possible not to empty the buffer when this occurred ? Otherwise an http check can be executed in the connect function.